src/handlers/proposalsHandlers.ts

```diff
@@ -1,11 +1,13 @@
 import {
+	assertProposalAuthorization,
 	createProposal,
 	getLimitValues,
 	loadProposal,
 	loadProposalBundle,
 } from '../database';
 import {
 	AuthenticatedRequest,
+	isId,
```

Comment:
I wonder if `isId` should set the maximum at the JavaScript maximum of 2^53-1? Then again, a postgresql ID can go higher than that. Except that we are using plain 32-bit ints, so it's lower. Hmm, I digress. It's OK.